app/src/main/java/de/s3xy/architecturesample/search/ui/SearchRepositoriesActivity.java

```diff
@@ -64,15 +70,43 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
     }
 
     @Override
-    protected void onDestroy() {
-        super.onDestroy();
-        mSearchPresenter.dettachView();
-        mUnbinder.unbind();
+    public boolean onCreateOptionsMenu(Menu menu) {
+        MenuInflater inflater = getMenuInflater();
+        inflater.inflate(R.menu.search_menu, menu);
+
+        if (mSearchPresenter.shouldShowSignIn()) {
```

Comment:
See previous comments, authentication should be handled in another presenter than Search